backend/dataall/modules/dataset_sharing/aws/ram_client.py

```diff
@@ -15,9 +15,8 @@ def __init__(self, account_id, region):
         self._account_id = account_id
```

Comment:
minor: this member isn't used anywhere now

Reply:
Opting to keep this as it could be used for extensions to the RamClient later on and matches the way we handle our other client classes